src/test/java/edu/mit/dos/object/ObjectServiceTest.java

```diff
@@ -34,7 +35,7 @@ public void setup(){
                 .setSingleView(view)
                 .build();
     }
-
+    @Ignore
```

Comment:
I think we decided that we need to keep this test?

Reply:
Oops, an oversight! I'll fix